src/components/number/number.jsx

```diff
@@ -55,12 +55,19 @@ function Number({
   const minimumFractionDigits = getFractionDigits(tickDecimals, valueMinDecimals, valueDecimals);
   const maximumFractionDigits = getFractionDigits(tickDecimals, valueMaxDecimals, valueDecimals);
   const formattedNumber = formatNumber(value, { minimumFractionDigits, maximumFractionDigits });
+  const absFormattedNumber = formatNumber(Math.abs(value), { minimumFractionDigits, maximumFractionDigits });
+  const ariaSign = (value < 0) ? 'minus ' : '';
```

Comment:
Should `'minus'` here be `'&minus;'` instead to match  https://github.com/nordnet/nordnet-component-kit/pull/597/files#diff-e93dc63fe4a992bb5e20dad504af1d55R12?